extensions/positron-r/src/r-installation.ts

```diff
@@ -56,7 +58,8 @@ export enum ReasonDiscovered {
 export enum ReasonRejected {
 	invalid = "invalid",
 	unsupported = "unsupported",
-	nonOrthogonal = "nonOrthogonal"
+	nonOrthogonal = "nonOrthogonal",
+	userExcluded = "userExcluded",
```

Comment:
Weakly held opinion: maybe just "excluded"? Motivated by pondering whether it's the user or an admin who has excluded it and I suspect more often it will be an admin. If you find this viewpoint compelling, it might also suggest change wording elsewhere, e.g. in logging. I guess one of the reasons this might actually matter is when a Workbench user is trying to figuring out their interpreter list and it could help them to know that an admin might have placed exclusions the user was not aware of.

Reply:
I find this compelling! I have updated the relevant places in both the R and Python extensions to remove "user" as you're totally right that it could be confusing both in logs and in the code.